src/Core/Addon/Theme/ThemeManager.php

```diff
@@ -294,6 +295,26 @@ private function doEnableModules(array $modules)
         return $this;
     }
 
+    /**
+     * Reset the modules received in parameters if they are installed and enabled
+     *
+     * @param string[] $modules
+     * @return $this
+     */
+    private function doResetModules(array $modules)
+    {
+        $moduleManagerBuilder = ModuleManagerBuilder::getInstance();
+        $moduleManager = $moduleManagerBuilder->build();
+
+        foreach ($modules as $moduleName) {
+            if ($moduleManager->isInstalled($moduleName) && $moduleManager->isEnabled($moduleName)) {
```

Comment:
Do you think we really need to check a module is enabled? What could be the consequence to not doing so?

Reply:
As an enabled module is "de facto" installed, I'd check only on enabled modules

Reply:
Indeed, we don't need to check both.
I prefer to check if the module is installed instead of enabled because if the user enable it later, it will be in the right state.